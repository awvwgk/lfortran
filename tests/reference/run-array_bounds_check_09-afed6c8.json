{
    "basename": "run-array_bounds_check_09-afed6c8",
    "cmd": "lfortran --no-color {infile}",
    "infile": "tests/errors/array_bounds_check_09.f90",
    "infile_hash": "a0763b8438157b6413dbda8eb34da783f57fcdb6f82be814b39f6a35",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "run-array_bounds_check_09-afed6c8.stderr",
    "stderr_hash": "a6667a156bcfea25baad9c09053df86046e9da2d9cc6795c4f9c32e0",
    "returncode": 1
}