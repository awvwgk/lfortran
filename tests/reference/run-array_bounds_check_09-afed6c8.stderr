At 12:9 of file tests/errors/array_bounds_check_09.f90
Runtime Error: Size mismatch in binary operation with operands '__libasr_created__integer_binop_' and '__libasr_created_integer_binop_right_'

Size of '__libasr_created__integer_binop_' is 3 and size of '__libasr_created_integer_binop_right_' is 2
